### PR TITLE
feat: Metadata update

### PR DESCRIPTION
**Pull request by jhomlala**
Changelog:
* Dart fix
* Metadata update

### CODE REVIEW

File: packages/alice/pubspec.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ description: Alice is an HTTP Inspector tool which helps debugging http requests
 version: 1.0.0-dev.9
 homepage: https://github.com/jhomlala/alice
 repository: https://github.com/jhomlala/alice
+topics: http, inspector, dio, chopper, interceptor, alice
```
Comment (techouse) [inline packages/alice/pubspec.yaml:6]: You can have a max of 5 and they need to be a list, i.e.
```yaml
topics:
  - http
  - inspector
  - dio
  - chopper
  - interceptor
```

File: packages/alice_chopper/pubspec.yaml [hunk 2]
```diff
@@ -1,8 +1,9 @@
 name: alice_chopper
 description: "Alice + Chopper integration. It contains plugin for Alice which allows to use Chopper package."
-version: 1.0.4
+version: 1.0.5
 repository: https://github.com/jhomlala/alice
 homepage: https://github.com/jhomlala/alice
+topics: http, inspector, alice, chopper, interceptor
```
Comment (techouse) [inline packages/alice_chopper/pubspec.yaml:6]: Same as above.

File: packages/alice_http/pubspec.yaml [hunk 3]
```diff
@@ -1,8 +1,9 @@
 name: alice_http
 description: "Alice + HTTP integration. It contains plugin for Alice which allows to use http package."
-version: 1.0.3
+version: 1.0.4
 repository: https://github.com/jhomlala/alice
 homepage: https://github.com/jhomlala/alice
+topics: http, inspector, alice, interceptor
```
Comment (techouse) [inline packages/alice_http/pubspec.yaml:6]: Same as above.

File: packages/alice_http_client/pubspec.yaml [hunk 4]
```diff
@@ -1,8 +1,9 @@
 name: alice_http_client
 description: "Alice + HTTP Client integration. It contains plugin for Alice which allows to use Http Client."
-version: 1.0.3
+version: 1.0.4
 repository: https://github.com/jhomlala/alice
 homepage: https://github.com/jhomlala/alice
+topics: http, inspector, alice, interceptor
```
Comment (techouse) [inline packages/alice_http_client/pubspec.yaml:6]: Same as above.

File: packages/alice_objectbox/pubspec.yaml [hunk 5]
```diff
@@ -1,8 +1,9 @@
 name: alice_objectbox
 description: "Alice + ObjectBox integration. It contains a plugin for Alice which stores http requests and responses in an ObjectBox NoSQL database."
-version: 1.0.0
+version: 1.0.1
 repository: https://github.com/jhomlala/alice
 homepage: https://github.com/jhomlala/alice
+topics: http, inspector, alice, interceptor
```
Comment (techouse) [inline packages/alice_objectbox/pubspec.yaml:6]: Same as above.

**Review by techouse — CHANGES_REQUESTED** [review]

File: packages/alice/pubspec.yaml (hunk 1 above)
Comment (jhomlala) [reply packages/alice/pubspec.yaml]: Yup, forgot about syntax.

File: packages/alice/pubspec.yaml [hunk 6]
```diff
@@ -3,7 +3,13 @@ description: Alice is an HTTP Inspector tool which helps debugging http requests
 version: 1.0.0-dev.9
 homepage: https://github.com/jhomlala/alice
 repository: https://github.com/jhomlala/alice
-topics: http, inspector, dio, chopper, interceptor, alice
+topics:
```
Comment (techouse) [inline packages/alice/pubspec.yaml:6]: It won't accept more than 5.

**Review by techouse — CHANGES_REQUESTED** [review]

File: packages/alice/pubspec.yaml (hunk 6 above)
Comment (jhomlala) [reply packages/alice/pubspec.yaml:6]: Yeah, I can't count 🤦

**Review by techouse — APPROVED** [review]
LGTM

**Pull request merged**